data/json/vehicle_placement.json

```diff
@@ -0,0 +1,107 @@
+[
+    {
+        "type" : "vehicle_placement",
+        "id" : "road_straight_wrecks",
+        "locations" : [
+            { "x": [0,19], "y": [0,19], "facing" : [90,270] }
```

Comment:
I assume this is supposed to declare where on the generated map the vehicle spawns. If so, it's probably not very useful without known the mapgen data.

E.g. if the mapgen defines a house that has a garage at the north, the vehicle should spawn there, but if the mapgen has the garage at the west side, it must spawn there. It is probably better to keep this directly with the mapgen stuff (which already supports spawning a vehicle).

Another problem is if somebody changes the mapgen data (e.g. moves the garage or the parking spot), they won't see this vehicle placement and will forget to update it accordingly.